docs/buildctl.md

```diff
@@ -0,0 +1,146 @@
+# buildctl
+
+`buildctl` is the command-line interface to `buildkitd`.
+
+```
+NAME:
+   buildctl - build utility
+
+USAGE:
+   buildctl [global options] command [command options] [arguments...]
+
+VERSION:
+   0.0.0+unknown
+
+COMMANDS:
+   du        disk usage
+   prune     clean up build cache
+   build, b  build
+   debug     debug utilities
+   help, h   Shows a list of commands or help for one command
+
+GLOBAL OPTIONS:
+   --debug                enable debug output in logs
+   --addr value           buildkitd address (default: "unix:///run/buildkit/buildkitd.sock")
+   --tlsservername value  buildkitd server name for certificate validation
+   --tlscacert value      CA certificate for validation
+   --tlscert value        client certificate
+   --tlskey value         client key
+   --tlsdir value         directory containing CA certificate, client certificate, and client key
+   --timeout value        timeout backend connection after value seconds (default: 5)
+   --help, -h             show help
+   --version, -v          print the version
+```
+
+## build
+
+Synopsis:
+
+```
+buildctl build --frontend dockerfile.v0 --opt target=foo --opt build-arg:foo=bar --local context=. --local dockerfile=. --output type=image,name=docker.io/username/image,push=true
+```
+
+`buildctl build` uses a buildkit daemon `buildkitd` to drive a build.
+
+The build consists of the following key elements:
```

Comment:
If we could use this as a table of contents and then link to each section below using `#` fragment links, that would be :ok_hand:

Reply:
Done, take a look.